experimental/Schemes/IdealSheaves.jl

```diff
@@ -555,6 +555,10 @@ function is_one(I::PrimeIdealSheafFromChart; covering::Covering=default_covering
   return false
 end
 
+function dim(I::PrimeIdealSheafFromChart)
+  return dim(I(original_chart(I)))
+end
+
 function is_one(I::SumIdealSheaf; covering::Covering=default_covering(scheme(I)))
```

Comment:
make this an attribute?

Reply:
No. It's already an attribute of the `original_ideal`.

Reply:
This is a `SumIdealSheaf`, what is the `original_ideal` ? There should be several.

Reply:
I thought you were referring to line 558.

Reply:
Line 562 is already an attribute. Look at the code below.

Reply:
Right :-) I was just looking for the `@attr`. But this methods takes several arguments and hence `@attr` does not work.